src/app/components/LiveLabel/__snapshots__/index.test.tsx.snap

```diff
@@ -48,9 +48,72 @@ exports[`LiveLabel should correctly render for RTL service 1`] = `
     role="text"
   >
     <span
-      aria-hidden="true"
       class="emotion-1"
       dir="rtl"
```

Comment:
do we need these dir's?

Reply:
We probably don't... the existing component did and we're trying to make it like for like. This is a Simorgh-wide problem (extra dir attributes where they are not needed), so I wonder if we could look into it separately (I have it on my todo list to remove all extra dir tags, to see if it would reduce page weight)...